Library/Homebrew/formula_installer.rb

```diff
@@ -242,15 +242,21 @@ def check_install_sanity
        # Integration tests override homebrew-core locations
        ENV["HOMEBREW_TEST_TMPDIR"].nil? &&
        !pour_bottle?
-      raise CannotInstallFormulaError, <<~EOS
-        #{formula}: no bottle available!
+      message = <<~EOS
+        #{formula}: unable to pour bottle!
```

Comment:
```suggestion
        #{formula}: no bottle available!
```

Let's leave this as-is for now and merge as-is and then can do more in a follow-up PR.